scripts/createConfig.ts

```diff
@@ -0,0 +1,225 @@
+        assert(
+          feeConfig.baseFee.numberOfPeriod *
+            feeConfig.baseFee.periodFrequency ==
+            TWO_HOURS
+        );
+        acc.totalExpoConfig += 1;
+      } else {
+        acc.withoutFeeSheduler += 1;
+      }
+
+      return acc;
+    },
+    {
+      totalDynamicConfig: 0,
+      totalLinearConfig: 0,
+      totalExpoConfig: 0,
+      totalQuoteOnlyConfig: 0,
+      totalBothTokenConfig: 0,
+      withoutFeeSheduler: 0,
+    }
+  );
+
+  assert(totalDynamicConfig == TOTAL_DYNAMIC_FEE_CONFIG);
+  assert(totalBothTokenConfig == TOTAL_BOTH_TOKEN_CONFIG);
+  assert(totalExpoConfig == TOTAL_EXPONENTIAL_CONFIG);
+  assert(withoutFeeSheduler == WITHOUT_FEE_SCHEDULER);
+  assert(totalLinearConfig == TOTAL_LINEAR_CONFIG);
+  assert(totalQuoteOnlyConfig == TOTAL_QUOTE_ONLY_CONFIG);
+
+  console.log("Success validate config");
+}
+
+export const TOTAL_LINEAR_CONFIG = 24;
+export const TOTAL_EXPONENTIAL_CONFIG = 24;
+export const WITHOUT_FEE_SCHEDULER = 24;
+export const TOTAL_DYNAMIC_FEE_CONFIG = 36;
+export const TOTAL_QUOTE_ONLY_CONFIG = 36;
+export const TOTAL_BOTH_TOKEN_CONFIG = 36;
+export const TWO_HOURS = 2 * 60 * 60;
+export const ONE_DAY = 60 * 60 * 24;
+
+(async () => {
+  const wallet = Keypair.fromSecretKey(
+    Uint8Array.from(require(configData.keypairFilePath))
+  );
+
+  const connection = new Connection(configData.rpcUrl);
+  const cpAmm = new CpAmm(connection);
+
+  const program = cpAmm._program;
+
+  validateConfig();
+
+  const dynamicFeeConfig = {
+    binStep: 1,
+    binStepU128: new BN("1844674407370955"),
+    filterPeriod: 10,
+    decayPeriod: 120,
+    reductionFactor: 5000,
+    variableFeeControl: 2000000,
+    maxVolatilityAccumulator: 100000,
+  };
+  // create
+  const result = [];
+  const errorIndex = [];
+
+  for (const feeConfig of configData.feeConfig) {
+    try {
+      const {
+        cliffFeeNumerator,
+        numberOfPeriod,
+        reductionFactor,
+        periodFrequency,
+        feeSchedulerMode,
+      } = feeConfig.baseFee;
+
+      const createConfigParams = {
+        index: new BN(feeConfig.index),
+        poolFees: {
+          baseFee: {
+            cliffFeeNumerator: new BN(cliffFeeNumerator),
+            numberOfPeriod,
+            reductionFactor: new BN(reductionFactor),
+            periodFrequency: new BN(periodFrequency),
+            feeSchedulerMode,
+          },
+          protocolFeePercent: 20, // 20% of lp fee
+          partnerFeePercent: 0,
+          referralFeePercent: 0,
+          dynamicFee: feeConfig.dynamicFee ? dynamicFeeConfig : null,
+        },
+        sqrtMinPrice: new BN(MIN_SQRT_PRICE),
+        sqrtMaxPrice: new BN(MAX_SQRT_PRICE),
+        vaultConfigKey: PublicKey.default, // default pubkey
+        poolCreatorAuthority: PublicKey.default, // default pubkey
+        activationType: 1, // default timestamp
+        collectFeeMode: feeConfig.collectFeeMode,
+      };
+
+      const configAccount = deriveConfigAddress(
```

Comment:
could you check if the configAccount has been initialized, so we can skip to send tx for that? 
if there is something happen, we can re-run the script

Reply:
updated.